_docs/setup/verify.md

```diff
@@ -80,8 +80,7 @@ protocol.
 
     1. **Windows**: Press Windows-R, type "powershell" and click OK.
     2. **MacOS**: Click the Searchlight (magnifying glass) icon in the menu bar, and
-    type a terminal window. "terminal". Select the Terminal application from the
-    search results.
+    type "terminal". Select the Terminal application from the search results.
```

Comment:
Ugh. This was correct in v0.91 doc. Another sloppy mistake made during the conversion to markdown.